types/canvasjs/index.d.ts

```diff
@@ -1,992 +1,1690 @@
-// Type definitions for CanvasJS v1.5.1
+// Type definitions for CanvasJS 1.9
 // Project: http://canvasjs.com/
-// Definitions by: Mark Overholt <https://github.com/mover5>
+// Definitions by: ShuYin Zhang  <https://github.com/brutalimp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace CanvasJS {
+export namespace CanvasJS {
```

Comment:
So, why i need both ` export = CanvasJS` and `export as name CanvasJS`, what are they used for?  Sometimes i saw `export  *  as namespace Somename` , what's the difference with  `export as name CanvasJS`? really thank you !